pkg/operation/botanist/component/metricsserver/metrics_server.go

```diff
@@ -350,8 +350,8 @@ func (m *metricsServer) computeResourcesData() (map[string][]byte, error) {
 									corev1.ResourceMemory: resource.MustParse("150Mi"),
 								},
 								Limits: corev1.ResourceList{
-									corev1.ResourceCPU:    resource.MustParse("100m"),
-									corev1.ResourceMemory: resource.MustParse("1Gi"),
+									corev1.ResourceCPU:    resource.MustParse("500m"),
+									corev1.ResourceMemory: resource.MustParse("2Gi"),
```

Comment:
Aren't those values too high? 
For single small node cluster this can occupy all resources.

PS. I think we should disable the VPA for the metrics server as we currently have a chicken-egg problem with them.

Reply:
Sure. I saw that the current limits for memory was `~900Mi` in the cluster in which the issue occurred, hence increased the memory limits too. But it is likely that CPU was the bottleneck there.

Shall I reduce the limits to `cpu: 500m, memory: 1Gi` then (to manage 500 nodes)?

Reply:
> Shall I reduce the limits to `cpu: 500m, memory: 1Gi` then (to manage 500 nodes)?

Sounds good to me.